cmd/wallet-web/src/pages/WACIIssue.vue

```diff
@@ -253,30 +187,25 @@ export default {
         return;
       }
 
-      // TODO [Issue#1336] read manifest, presentations, normalized, comment, fields to enhance UI
+      // TODO: issue1336 - read manifest, presentations, normalized, comment, fields to enhance UI
       this.interactionData = { threadID, fulfillment, domain, challenge, error };
     } catch (e) {
       this.handleError(e);
       return;
     }
 
-    this.credentialDisplayData = await credManifest;
-
     // prepare cards
     this.prepareCards();
 
     // get all vaults
     const collectionManager = new CollectionManager({ agent: this.getAgentInstance(), user });
-    this.fetchAllVaults(token, collectionManager);
+    await this.fetchAllVaults(token, collectionManager);
```

Comment:
fetching all vaults also shouldn't block UI rendering of the page. Did you run into any issue because of that?

Reply:
Addressed